content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -378,12 +384,34 @@ Run `GameLauncher.exe` from your project game release layout, which is located i
 
 - `C:\MyProject\install\bin\Windows\release\Monolithic` -- For monolithic projects.
 
-Now you can distribute your project to other Windows devices. To run the Game Launcher, you must install the [Microsoft Visual C++ (MSVC) Redistributable](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-160).
+
+## Distribute your build
+
+To distribute your build to other Windows computers:
+
+1. Create a `.zip` file of the `<install>\bin\Windows\release\<build>` folder. The contents of the `.zip` file should look similar to this:
+
+        Monolithic
+        |   DevTestProject.GameLauncher.exe
+        |   PhysXDevice64.dll
+        |   PhysXGpu_64.dll
+        |   
+        \---Cache
+            \---pc
+            engine.pak
+                
+1. Distribute the `.zip` file using your preferred method, such as a file storage service. 
+
+Now others can download and unpack the `.zip` file and run the Game Launcher on their Windows computers.
+
+{{< note >}}
+To run the Game Launcher on the other Windows computers, they must have the [Microsoft Visual C++ (MSVC) Redistributable](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-160) installed.
+{{< /note >}}
```

Comment:
@paucom  I formatted the text in a note as you suggested. I also added the sentence "Now others can download..." to conclude the steps in this section.